### ISSUE TITLE
latest version wordpress and latest version plugin  Call to undefined function is_plugin_active()

### ISSUE THREAD

**Issue by ramvyr** [digitoimistodude/air-helper#43]
Fatal error: Uncaught Error: Call to undefined function is_plugin_active() in mail-delivery.php:25

**Comment by timiwahalahti** [comment]
Hi @ramvyr!

Can you provide some more information, the `is_plugin_active` is [WordPress core function](https://developer.wordpress.org/reference/functions/is_plugin_active/) and should be always available. We've also tested the latest version (2.10.2) on multiple sites and haven't encountered the issue you are describing.

* Which version of WordPress are you running?
* Is the installation/environment setup you are using somehow special or is it more or less standard?

**Comment by ramvyr** [comment]
Hi, @timiwahalahti.
I used windows and openserver.
Recently at other project, on the same assembly I used 2.8.0 version plugin and there were no problems

**Comment by ronilaukkarinen** [comment]
@ramvyr Are you sure your **WordPress installation** is [5.8](https://wordpress.org/download/)?

> Which version of WordPress are you running?

Re-tested just now with WordPress 5.8 and Air-helper 2.10.2. Should work properly.

**Comment by ramvyr** [comment]
> Re-tested just now with WordPress 5.8 and Air-helper 2.10.2. Should work properly.

I used wp 5.8 and 2.10.2 plugin.

I made wrapper
```
if (function_exists('is_plugin_active')) {
}
```and all works

**Comment by timiwahalahti** [comment]
With this relatively limited information and the fact that we aren't seeing the same issue in production or neither after testing it few times with our own stack (dudestack) and normal installation, I'm going to close the ticket.

If you find more information about the issue and are able to provide more details, feel free to re-open the issue.

**Comment by ronilaukkarinen** [comment]
I was able to reproduce this issue with newest dudestack, air-helper and air-light. I have no idea what causes it. Reopening.

**Comment by timiwahalahti** [comment]
@ramvyr Sorry that fixing this did take so long! It's now resolved in commit 387253c91a712fab61960f486feef8853fa9c957 which will be included in the 2.11.0 release coming out shortly.
